docs/pages/guides/schema-validation/syntax.mdx

```diff
@@ -243,6 +243,20 @@ root.set(
 root.set("shapes", new LiveMap([["shapeId", { x: 100, y: 100 }]]));
 ```
 
+## Unions
+
+You can model a choice between two types using a union, which will be familiar
+from TypeScript. Here are some examples:
+
+```ts
+type Storage {
+  ids: (string | number)[]
+  selectedIds: string | null
+  person: LiveObject<Person> | null
+  people: LiveList<LiveObject<Person>> | null
+}
+```
```

Comment:
@taylorpwhipp What do you think? Examples make sense? Too complicated? Feel free to come up with simpler examples! 🙏

Reply:
makes sense to me!